week-1/L-functions-nested/exercise2.js

```diff
@@ -0,0 +1,17 @@
+var mentor1 = "Daniel";
+var mentor2 = "Irina";
+var mentor3 = "Mimi";
+var mentor4 = "Rob";
+var mentor5 = "Yohannes";
+
+function mentors ( names) {
+    return names.toUpperCase(names);
```

Comment:
There's a little bug here. Once you've googled the method definition of `toUpperCase` can you spot it?